scripts/xtensa-build-zephyr.sh

```diff
@@ -11,7 +11,7 @@ SOF_TOP=$(cd "$(dirname "$0")" && cd .. && pwd)
 DEFAULT_PLATFORMS=()
 
 # Intel
-DEFAULT_PLATFORMS+=(apl cnl icl tgl-h tgl)
+DEFAULT_PLATFORMS+=(apl cnl icl jsl tgl-h tgl)
```

Comment:
Depending on how long the Zephyr PR is going to take, you can add it to `SUPPORTED_PLATFORMS` right now. This will enable other people to try it now without breaking CI.

Same thing with line 84 in the python script.